packages/form/addon/components/cf-field/label.hbs

```diff
@@ -3,11 +3,18 @@
     {{@field.question.raw.label}}
   </span>
 
-  {{#if (and (not @field.question.isCalculated) @field.optional)}}
-    <span
-      class="uk-margin-small-left uk-text-muted uk-text-lowercase uk-text-normal"
-    >({{t "caluma.form.optional"}})</span>
-  {{/if}}
+  {{#unless @field.question.isCalculated}}
+    {{#if (and @field.optional (not this.useMandatoryAsterisk))}}
+      <span
+        class="uk-margin-small-left uk-text-muted uk-text-lowercase uk-text-normal"
+      >({{t "caluma.form.optional"}})</span>
+    {{else if (and (not @field.optional) this.useMandatoryAsterisk)}}
+      <span
+        title={{t "caluma.form.mandatory"}}
+        class="uk-text-muted uk-text-lowercase uk-text-normal mandatory-asterisk"
+      >* </span>
```

Comment:
Is the whitespace afterwards intended?

Reply:
Was intended, yes. To make it more comfortable for the user, who's trying to hover it to display the title. If you find it cleaner - I easily can remove the space (should be fine)

Reply:
Fair enough, let's keep it.